example-basic.md

```diff
@@ -0,0 +1,16 @@
+---

```

Comment:
I think you meant to edit files in `docs/`, but the changes here are *adding* files to the root (`/`) directory of the repo instead.